cleanlab/count.py

```diff
@@ -425,20 +400,12 @@ def compute_confident_joint(
     Parameters
     ----------
     labels : np.ndarray
-      An array of shape ``(N,)`` of noisy labels, i.e. some labels may be erroneous.
-      Elements must be in the set 0, 1, ..., K-1, where K is the number of classes.
-      ``len(set(labels)) == pred_probs.shape[1]`` for standard multi-class classification with single-labeled data (e.g. ``labels =  [1,0,2,1,1,0...]``).
-      For multi-label classification where each example can belong to multiple classes(e.g. ``labels = [[1,2],[1],[0],..]``),
-      your labels should instead satisfy: ``len(set(k for l in labels for k in l)) == pred_probs.shape[1])``.
+      Given class labels for each example in the dataset, some of which may be erroneous,
+      in same format expected by :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` function.
```

Comment:
If we redefine the labels parameter in num_label_issues then links like this could be more localized.